### ISSUE TITLE
Stun refactoring

### ISSUE THREAD

**Issue by ComicIronic** [d3athrow/vgstation13#3594]
I hate current stuns.
Deciding an encounter with stun weaponry or an RNG push is boring. It makes any fight that isn't a properly-planned ambush far more boring than it should otherwise be, because the entire fight revolves around simply avoiding a single stun bullet and hoping your opponent doesn't avoid yours. While I'm sure people would argue there's some skill in this, it's about as interesting to play as watching paint dry, since actually getting stunned is immediate death with no chance of recovery or further combat through grappling or the like.

The power of stun weaponry also leaves crewmembers effectively defenseless against a mildly-armed antag who wants to murderbone, which is well-known as one of the most dull things that can happen in a traitor round for the players involved.

tl:dr; I think stuns are too powerful and need fixing so the game is actually more fun.

**Comment by SkowronX** [comment]
Change we can believe in.

**Comment by clusterfack** [comment]
I happen to agree completely. 

If we are brainstorming right now, I think we should migrate to a two-factor system of health where pain and damage have relatively equal priority. Pain would serve to disable limbs and slow people down, and stuns as they stand right now would function based on inflicting much pain to a player, particularly the targeted limbs, disabling those limbs to either slow down or drop a weapon, possibly blind momentarily, etc. Right now our current 'pain' system is just dropping thing because your hand is broken, and getting spammed by messages in the chat log about how it hurts.

**Comment by AutismSpider** [comment]
How about stamina system like goon has?

Generally speaking, each direct interaction with another player (defensive or offensive) drains stamina, which will be replenished automatically at a slow rate. Once your stamina reaches 0 or less, you are unable to talk and likely to be knocked down by another attack, i.e. unable to defend yourself.


Aka it take X taser shots to down some one.

Stuff like Ebow Stun baton etc = auto stun unless high on pain meds etc.

**Comment by 9600bauds** [comment]
I don't think I like the "pain" idea, since tasers and such aren't really pain and moreso electric shock cramping you to all fuck. (plus having to get painkillers for the innocent guy you tased would be really tiring)

I hear Bay has a good system in place, I've never been there though, can anyone compare?

**Comment by Uqualm** [comment]
You are literally going to ruin the whole game if you change this.

**Comment by ComicIronic** [comment]
I didn't know my life goal was so easily achievable.
Cancel the PRs, I'm crashing this game with no survivors.

**Comment by Uqualm** [comment]
Let us consider the possibility of having to shoot someone twice with a taser shot. What are the immediate effects of this? Well for starters, it is literally twice as hard for a security officer to do his job. 

You are a security officer and you see an assistant hacking into the teleporter. As soon he sees you he runs away. You chase after him, he's in your sight, you fire one taser shot. It misses. 4 shots left. You continue to chase. You fire two more shots. One hits. One misses. Because it now takes two stuns, he shrugs it off and runs away into maintenance. You follow him into maintenance. Two shots left. You fire one more shot, and it hits. You finally have caught the guy one shot left for you taser. You cuff the guy and start dragging him to the brig. Still in maintenance, a man with a gas mask and an energy sword comes at you. You draw your taser and fire a shot at him. It hits, but there's no affect. He hits you once with an E-sword, you are knocked down. You are chopped in the head with an e-sword. You enter crit. You die.

And that's just with traitors. I could go on and on and on (I think I will actually!) It's pretty important for you to realize just how retarded and out of touch you are.

**Comment by ComicIronic** [comment]
You make a lot of assumptions.
You assume, for example, that a single taser shot would somehow have no effect. You also assume that it would be possible to simply "shrug off" a taser shot. You then assume that a sec officer is too incompetent to switch to a stun baton when their taser is out of charge.

All pretty invalid assumptions, I should add.

**Comment by Uqualm** [comment]
I can only make assumptions because you have offered no solution to the supposed problem. You have not said any details on how a rework of the system should work. Just "IZ BAD CUZ I DON"T LIKE IT". You still have not answered my question. What do you instead suggest?

**Comment by ComicIronic** [comment]
That's because the entire issue is dedicated to solving the problem, nor do I think it valid criticism to object to a system that you also don't know the details of for exactly that reason.

This is a place to bring forward ideas, not to attack a non-existent proposal that you dislike the self-made features of.

>no solution
>several suggestions and discussions above your post

I am personally a fan of Skowron's suggestion, which could use pain in interesting ways to allow sec officers to use tasers more effectively. It would require sufficient pain to the chest to knock someone over, for example, but targeting their legs could slow them down or targeting their hands could disarm them. This also has the benefit of integrating well with disarm pushes, since it would no longer be possible to just shove and fighting would involve real fistwork.

If you really have something to say against this, I suggest you more focus on why the system shouldn't change, which is something I've not seen a convincing argument for.

**Comment by Uqualm** [comment]
Wow, I thought maybe I was wrong, but I really am right. You coders are so far up your own asses it's not even funny. Any change to the stun system is going to result in a change that makes it so it takes multiple taser shots to stun someone. You cannot dispute this, as it is the reason you want the change in the first place. It is quite obvious to me you do not play the game, much less security or any sort of antagonist role. What you are doing by suggesting any sort of change to the current stun system is going to completely ruin the fragile balance that has been established between security and the antagonists. You say that "The power of stun weaponry also leaves crewmembers effectively defenseless against a mildly-armed antag who wants to murderbone" which is a bunch of nonsense. If it is as you say and "actually getting stunned is immediate death with no chance of recovery" how are they defenseless? Due to the nature of the current stun system, anyone can take down someone who is murdering a bunch of people, all it takes is one shot. By taking away Security's only legitimate defense against antagonists you will only be exacerbating this mild problem and turning it into a major one. Security for the most part is armed only with tasers, batons, and flashes. You want to nerf these non lethal methods of dealing with threats because it's "not interesting" but I can assure you that having security and the crew being steam rolled by the antagonists is much less interesting.

Let's pretend you have your way and you get a "pain system" where if you are in pain, various affects happen, one of which being a full stun. You want the non lethal weapons to operate on this principle of causing pain and not damage. But what you fail to realize is that lethal weapons cause pain as well, unless you mean to tell me being shot in the chest with a bullet or laser is painless. This means that the effects of the pain, and the actual damage of the lethal weapon now stack. For one consider the most common and benign antagonist, the traitor. Of the items that can cause damage that a traitor employs are as follows: the revolver, energy crossbow, and energy sword. In its current state the revolver does 60 brute damage per shot, and frequently causes internal bleeding. This is already enough to effectively kill someone in two shots. With the proposed pain addition, it now has the added benefit of stunning on a successful hit. The energy crossbow is put into a scenario a vast majority of stun weapons would have to face. You can opt to remove it's stunning ability, making it effectively useless, you can keep it's stunning effect that will stack with the pain that it inflicts via damage, or you can remove the damage it causes completely effectively turning it into an old taser. The problems with the energy sword are similar to those that the revolver face. It is a high damage weapon which means it causes lots of pain, it is unique in the fact that it can deflect taser shots and already has a high knockdown (stun) chance. This would make the energy sword a perfect weapon for a murderboning traitor. So, that covers all the damage dealing traitor weapons. These weapons are already in extremely high use, I imagine with the overall buffs they receive they would be even more popular. 

But this is just considering traitors! Do not even get me started on Nuke Op's with their 20 brute damage, internal bleeding causing, insta-stun guns, wizards with their Magic Missile, changelings with ranged para and death stings, etc. The overall advantages that antagonists would receive from this change are enormous and the immediate affects would be an incredibly deadly antag and an (actual) defenseless crew.  Why I have to bother to explain this to you is beyond me, it should be obvious. You'd have to be an out of touch coder to not realize the many problems a change in the stun system, the system the entire game is based on, would cause. And this is just off the top of my head, there are most likely many more unforeseen changes that would result of this, mainly changes within the meta of the game. I suppose you wouldn't know anything about that though, considering you're just an out of touch coder.

**Comment by SangriaMonkey** [comment]
Unless you want to completely change the combat system, and I mean everything in it, this won't work out all that well. Combat is basic but there is a little nuance involved. Even when cuffed or stung by a ling there is still a chance for you to get away under the right circumstances. *Plus if you make basic security things even harder you're going to have about one HoS or detective per day at best.*

**Comment by Senaat1** [comment]
My opinion is that before changing anything, think of the effect it's going to have on Security. Security is meant to be in a position of power, and Security has enough trouble dealing with chucklefucks as it is. Look at it from Security's POV, can they do their jobs effectively and stop greyshirts and antags. If anything make it a small nerf in the way of there being a highly-restricted chemical requiring diamond and plasma that will give temporary stun immunity. Of course use of this chemical or even possession of it is grounds for an execution, no questions asked.

**Comment by Boxe** [comment]
add the tator implant from /tg/ that lets you get out of one stun

**Comment by dylanstrategie** [comment]
I'm going to respond to that wall of text because no-one else will do it

>Wow, I thought maybe I was wrong, but I really am right.

Nice starter

>You coders are so far up your own asses it's not even funny.

Says the person who states in the very next line : 

>Any change to the stun system is going to result in a change that makes it so it takes multiple taser shots to stun someone. You cannot dispute this, as it is the reason you want the change in the first place.

>It is quite obvious to me you do not play the game, much less security or any sort of antagonist role.

Implying implications. I play the game a lot, but tapping text in a box and then compiling and running that text to make sure shit isn't broken doesn't allow me to get my valids 24/7

>What you are doing by suggesting any sort of change to the current stun system is going to completely ruin the fragile balance that has been established between security and the antagonists. 

There's no such thing as a fragile balance between Security and Antagonists. Either Security is staffed by a lot of well-learned players and the antagonists get shit-stomped, or Security is near empty and the first greyshit to obtain some form of weaponry (even from a lucky disarm) becomes the king of the station

>You say that "The power of stun weaponry also leaves crewmembers effectively defenseless against a mildly-armed antag who wants to murderbone" which is a bunch of nonsense.

It makes a lot of sense to me. If ANYONE, much less an antagonist, tasers you with the obvious intent to neutralize or kill you, you might as well tab out and go  take a walk, because you're not going anywhere until they find your body again

>If it is as you say and "actually getting stunned is immediate death with no chance of recovery" how are they defenseless? Due to the nature of the current stun system, anyone can take down someone who is murdering a bunch of people, all it takes is one shot.

Not everyone has ranged stun weaponry, and if almost every weapon on station is obsoleted by a taser gun because it can land a guaranteed neutralization, there is a problem

>By taking away Security's only legitimate defense against antagonists you will only be exacerbating this mild problem and turning it into a major one. Security for the most part is armed only with tasers, batons, and flashes.

The goal is to nerf stunning, not remove it. Also, stun batons are a valid close-range neutralization weapon, and for dangerous personnel there's lethal weaponry in the Armory, which no-one uses (except for the egun, for obvious reasons) specifically because there's no stun function. You can run from a lucky laser hit, you can't run from a lucky taser hit

>You want to nerf these non lethal methods of dealing with threats because it's "not interesting" but I can assure you that having security and the crew being steam rolled by the antagonists is much less interesting.

Implying implications again. Any fight being over when someone wipes out a taser isn't interesting indeed, but nerfing tasers does NOT, under any circumstances, open the entire station to antagonists. If anything, this makes the antagonist's job harder since they no longer have a valid way of quickly downing multiple people trying to put them down (which can currently be done with an egun, with minimal effort)

>Let's pretend you have your way and you get a "pain system" where if you are in pain, various affects happen, one of which being a full stun. You want the non lethal weapons to operate on this principle of causing pain and not damage. But what you fail to realize is that lethal weapons cause pain as well, unless you mean to tell me being shot in the chest with a bullet or laser is painless.  This means that the effects of the pain, and the actual damage of the lethal weapon now stack.

Yes, lethal weaponry would cause pain too

>For one consider the most common and benign antagonist, the traitor. Of the items that can cause damage that a traitor employs are as follows: the revolver, energy crossbow, and energy sword. In its current state the revolver does 60 brute damage per shot, and frequently causes internal bleeding. This is already enough to effectively kill someone in two shots. With the proposed pain addition, it now has the added benefit of stunning on a successful hit.

You are completely missing the point. For one, this is a GOOD thing that a Traitor would remotely consider the Revolver and the Energy Sword viable alternatives to other equipment, since right now they're flamebait and newbie bait. Then, getting nailed in the chest with a Revolver will ground you in a few second's time unless you literally have a super-medicine shot ready in your pocket, and the Traitor can -and most likely will- shoot again to crit you, before you can even react

>The energy crossbow is put into a scenario a vast majority of stun weapons would have to face. You can opt to remove it's stunning ability, making it effectively useless, you can keep it's stunning effect that will stack with the pain that it inflicts via damage, or you can remove the damage it causes completely effectively turning it into an old taser.

This is not up to me, unless I end up being the one performing that rework, but clearly the energy crossbow should not keep its instant stun on hit. I would rather have it fuck up the target with massive amounts of radiation, and keep it as a stealth weapon (it currently generates no attack message and is extremely discreet, and while your target will know, they're in a tight spot)

>The problems with the energy sword are similar to those that the revolver face. It is a high damage weapon which means it causes lots of pain, it is unique in the fact that it can deflect taser shots and already has a high knockdown (stun) chance. This would make the energy sword a perfect weapon for a murderboning traitor.

The energy sword deflecting shit is not a problem, if it need tuning down, sure, but it's mostly in as kind of a cool thingie for people who are aware of the inner workings of items. As for the pain, you're not walking off an energy sword assault, though this certainly doesn't prevent you from trying to hit the Traitor yourself, or just fucking running

>So, that covers all the damage dealing traitor weapons. These weapons are already in extremely high use, I imagine with the overall buffs they receive they would be even more popular. But this is just considering traitors! 

That has to be the funniest thing I've read today. Last I checked, parapens, C4 and emags completely eclipsed the Revolver and Energy Sword, which rely 100 % on absolutely everyone on board being too retarded to try and taser you, or getting stealth kills on unarmed crewmen. This change would only make them marginally viable if you're not shit at fighting, and even then this implies that we'd be completely removing tasers disabling people, which we are not

>Do not even get me started on Nuke Op's with their 20 brute damage, internal bleeding causing, insta-stun guns

The number 1 cause of failure for Loud Ops is not having millisecond reflexes and getting tased, either by the literal flood of crewmen with tasers and eguns or by borgs. And CR20s give you a large window of time to react and fight back while tasers put you out of the fight for 10 seconds, or forever if anyone can even bother

>wizards with their Magic Missile

Wizards are supposed to be overpowered since they are a one-man-army antag by design

>changelings with ranged para and death stings, etc.

Death stings have nothing to do with stuns and are delivered in a stealthy manner. Ranged para costs precious genomes and allows you to stun from three tiles away, which is of dubious use. Changelings in general need paralyzing abilities to work

>The overall advantages that antagonists would receive from this change are enormous and the immediate affects would be an incredibly deadly antag and an (actual) defenseless crew.

Factoring everything above, fuck no. This makes it a bit easier for antags, of course, but then Security will actually get robust and use some real fucking weaponry, or will learn to not just stand in the middle of a hallway soaking bullets and lasers

>Why I have to bother to explain this to you is beyond me, it should be obvious. You'd have to be an out of touch coder to not realize the many problems a change in the stun system, the system the entire game is based on, would cause.

That claim is ridiculous, SS13, at its absolute base, was designed as a space station simulation with atmospherics in mind. Furthermore, we are very in touch. You cannot code for this game if you have no idea what you're working with, and that on a very functional level

>And this is just off the top of my head, there are most likely many more unforeseen changes that would result of this, mainly changes within the meta of the game. I suppose you wouldn't know anything about that though, considering you're just an out of touch coder.

We know what we are doing. The amount of time where people have burst in holy rage due to us changing major systems and claiming everything was over, and subsequently, within a week or so, have gotten used to the changes and developed a new meta, are countless. No-one here is out of touch but you, for claiming all of this

>Plus if you make basic security things even harder you're going to have about one HoS or detective per day at best.

If people don't want to play Security, it's not something we'll solve by holding their hands and making it so easy they might as well press a button every five minutes to valid all living antags

>add the tator implant from /tg/ that lets you get out of one stun
>If anything make it a small nerf in the way of there being a highly-restricted chemical requiring diamond and plasma that will give temporary stun immunity.

If pain is added, painkillers will allow this, although they will -obviously- cause overdose if you swallow metric tons of it, so no swallowing 500 units of tramadol and walking around like you are Superman

**Comment by SangriaMonkey** [comment]
I implore you to play clown for a week and tell me that a lucky stun shot is a death sentence.

**Comment by Senaat1** [comment]
UQualm is Sarca, Dylan, don't waste too much time arguing with him.

My issue with rebalancing tasers is that everyone seems to be arguing from the belief that everyone should be on equal grounds, whether they're the Librarian or Security, and that having one party be overpowered is inherently a bad thing. 

I disagree on that. 

Security is meant to be in a position of power, at least at the start of the round. They get the guns, they get the armour, they're the ones supposed to be keeping the law and they should have the means to do so. If you're a traitor or a greyshit, better make sure you avoid Security until you're on equal grounds to fight them. If your argument is "well this traitor cargotech can get his hands on tasers and eguns no problem, and then i get tazed and instantly lose", stuns give you a lot of time to shout over the intercomms for help and if you have your suit sensors on, your corpse will be found and Security can deal with the person that killed you, which is their job.

And even so, acting as if a fight is over the instant a taser is whipped out is a gross oversimplification on your part. There is a relatively high chance you will dodge a taser shot already, so even in a one-tile corridor you stand a chance, and in a regular hallway there is plenty of room to move around and dodge taser shots.

And because i don't know how to quote you because i am a computer wizard, on the topic of traitor items: Eswords blocking tasers (especially double eswords) is a HUGE deal and can make tasers almost entirely useless if the traitor knows how to move as well, considering the amount of shots that would be blocked and the amount of shots he could dodge. on top of the innate dodge chance.

If you'd have to get two shots in, or multiple shots in general, it'd be damn near unbeatable.

Also Dylan, 'cause i don't know how to into code at all, do you have any idea what the exact blockchance on 2blade is?

**Comment by ComicIronic** [comment]
>lethal weapons cause pain

Which, funnily enough, is what happens in real life, because most people don't walk away from shootings (unless it's 9mm). This is a good thing, since current lethal methods get no love for being way less effective than the simple sidearm of any regular sec officer and a baton. 
Nobody said a lethal weapon had to be as good as a taser for pain, of course.

>Due to the nature of the current stun system, anyone can take down someone who is murdering a bunch of people, all it takes is one shot. By taking away Security's only legitimate defense against antagonists you will only be exacerbating this mild problem and turning it into a major one.

That's ridiculous. Nearly every job has no ranged stun that isn't either "get a stun gun from armory/cargo/rnd" or a ridiculous powergaming move to have prepped already. The biggest issue in lowpop murderboneing is exactly that i.e. the lack of stuns against a stun-wielding traitor in a stun-decided combat system.

>What you are doing by suggesting any sort of change to the current stun system is going to completely ruin the fragile balance that has been established between security and the antagonists. 

You got me. You know the truth. There is only one possibly balanced system per server, and that's our current one. I was hoping to ruin combat by sneaking the change under everyone's noses.

Or, perhaps, we can - shock horror - rebalance the system. Sure, there'll be growing pains, but I think that issue is eclipsed by the current issue with stuns themselves.

>My opinion is that before changing anything, think of the effect it's going to have on Security. Security is meant to be in a position of power, and Security has enough trouble dealing with chucklefucks as it is. 

While I appreciate that, there is something called rebalancing. Reducing taser effectiveness does not make tasers useless, and the average Joe Schmoe probably won't have immediate preparations against multiple taser shots. The rework would be more in favour of those prepared for a fight.

**Comment by SangriaMonkey** [comment]
>the average Joe Schmoe probably won't have immediate preparations against multiple taser shots

It's called "serpentine down a hallway" which *anyone can do*.

**Comment by ComicIronic** [comment]
>It's called "serpentine down a hallway" which anyone can do.

And which would intentionally be harder after being hit with a taser bolt, because of the pain system; this is also an issue with the current ranged stuns, only getting lucky is so much more important.

**Comment by SangriaMonkey** [comment]
Please play clown. Please. Unless one landed shot makes you treeman-tier slow you can still get most officers to empty their four shots no problem.

**Comment by Senaat1** [comment]
Comic, if someone can't move they might as well be stunned. What're they going to if they're slowed down? Turn around and also taze the person tazing them? Because in that case, it doesn't matter considering the initial tazer already has one taze off while the tazed one needs to taze twice.

The one defense against tazing is movement, being slowed down is effectively the same as getting knocked down.

**Comment by ComicIronic** [comment]
>Comic, if someone can't move they might as well be stunned. What're they going to if they're slowed down? Turn around and also taze the person tazing them? Because in that case, it doesn't matter considering the initial tazer already has one taze off while the tazed one needs to taze twice.

Or use another gun, or a thrown weapon, or escape through a door, or respond in a way that doesn't involve keeling over immediately. Nobody said it had to be 50/50 after the first tase, but the goal is to give the ability to respond in some way, even if that means having a slowdown. 

But attempting to remove an opponents weapon risks not getting as high a slowdown, so combat becomes more nuanced.

>Please play clown. Please. Unless one landed shot makes you treeman-tier slow you can still get most officers to empty their four shots no problem.

Which is just a symptom of the current system and its issues. Tasers are small-clip because stuns are so effective. I don't care if tasers are easy to dodge, it's getting hit with them that I care about.

**Comment by Probe1** [comment]
I have played the game for a little under a year.

When I was new to vg (and ss13) I did not like stuns.  They made me mad.  I lost fights because of one false step.  

But after months of playing I became more skilled and I can dodge taser shots easier.  There are still many difficult things, like chloral and other stuns that don't give you an obvious clue like a moderate speed projectile coming at you.  However they can be mitigated by having good game sense and reading situations (and people).

Now I'm on the other side of the fence.  I'm the one stunning new players and they're the ones crying pls nerf.  

If you want to remove or nerf stuns that's fine by me.  But it's going to take rebalancing all of combat.  All of it.  Which is a huge job.  If someone were to just nerf stuns it would make the game much worse.  It's all or nothing and I don't see anyone picking up the mantel to rebalance the whole fucking combat of the game.

If you want to take every single antag and fix their items, plus quite a few departments and rebalance so much, please do it.  It would be interesting and novel to have stuns weakened.  But just nerfing stuns will fuck the game up and waste everyones time.

**Comment by SangriaMonkey** [comment]
>give the ability to respond in some way

But you can easily get up and do just that if they aren't ready with cuffs with the current system. Every fight is different and even with the best gear there is still a chance that someone will fuck up somewhere. The game is dynamic like that. With multiple stuns needed people will just spam lasers or shotshells down hallways, or end up doing https://www.youtube.com/watch?v=KuYd9pA8LIY&t=39s with stun weapons if they're dipshits. So unless you're hell bent on redoing about eight years of work to make the game easier for yourself I would reconsider.

**Comment by ComicIronic** [comment]
>But you can easily get up and do just that if they aren't ready with cuffs with the current system.

Unless they choose to stun you while you're down. And since there's a readily-available way of getting cuffs in the first place, I don't think this is a particularly good response.

>So unless you're hell bent on redoing about eight years of work to make the game easier for yourself I would reconsider.
>to make the game easier for yourself

Why is this the immediate assumption?
Why is the combat system some sort of diamond in the rough of shitcode that turned out perfect, rather than a recognisably flawed and failing system that should have been redone alongside everything else that was more than 5 years old?

It's a telling thing that every major codebase but us has changed stuns in some way. They're the most obvious remnant of real oldcode.

**Comment by SangriaMonkey** [comment]
>I don't think this is a particularly good response

It happens. I've been on both sides countless times and let me tell you, slip ups do happen. And they're hilarious. You should really play the game some more before *demanding* changes to things. Stuns aren't an end-all-be-all thing, especially when you can get behind some glass with a laser, shoot your attacker with an ion weapon, set off an EMP, slip them, hide in a locker, run into maint, run into a crowd, yakety sax around the bridge, jab them with chloral, or otherwise render their shitty little zappy gun ineffective.

**Comment by Cutelilduck** [comment]
Thank you all for holding this debate. I hope everyone keeps a clear mind.
I'm going to drop some buzzwords:
1. Diminishing returns
2. Cable Cuffs nerf in decuffing time or something else.
3. Armor/Clothes protecting against stuns, weak to EMP.

**Comment by Uqualm** [comment]
>There's no such thing as a fragile balance between Security and Antagonists. 
Oh really? Are you sure? I know that you are new and probably do not remember, but there have been plenty of times where one antagonist role has had advantages that skewed the odds in their favor. Malfunctioning AI for example, was nearly impossible to stop.
>Either Security is staffed by a lot of well-learned players and the antagonists get shit-stomped, or Security is near empty and the first greyshit to obtain some form of weaponry (even from a lucky disarm) becomes the king of the station
Although you are wrong here, what you are saying is proof of exactly what I am saying. If there was NOT balance, then it would not matter how skilled a player is. One side would have more of an advantage then another. There could be no "either", only an always. The way it is now, they (and by they, I use the model antagonist, the traitor) and security are effectively equal.
>It makes a lot of sense to me.
This is something I've noticed you and the other brain dead coder (whose comment I will deal with next) do throughout your comments. You take two concepts that are directly linked to each other, separate them (removing them from the appropriate context) and then treat them as if they are two separate ideas. It does not make sense. I go on to explain why after that. Moving on to your response to my actual explanation.
>Not everyone has ranged stun weaponry, and if almost every weapon on station is obsoleted by a taser gun because it can land a guaranteed neutralization, there is a problem
You are right. Not everyone has access to ranged stun weaponry. But not every antagonist has access to ranged stun weaponry as well. Tell me, if they can acquire a weapon why can't you? If you're in medical, you can make a syringe full of chloral. If you're in science you actually CAN make guns! If you're in cargo, you can ORDER guns. If you are a civilian, you can make cable bolas. If you are an engineer, you can steal weapons just as they most likely have.

This is of course only accounting for RANGED stuns. Most have access to regular stuns in general such as stun prods, stun gloves, etc.

>The goal is to nerf stunning, not remove it.
You are not removing stunning, you are removing the only realistic defense that Security has against antagonists which is an instant stun. I would really love for you to experience the SS13 world you say you would like to live in. Please, go to /tg/ right now, login and pick security officer. On /tg/, the taser has been replaced with a hybrid taser, which has a second firing mode. This second firing mode effectively stuns in multiple shots. Here's a challenge I'd like for you to do. Try to play a security officer for a couple rounds without using the regular tase function, only use the disabler. I'd LOVE to know how that goes.

>but nerfing tasers does NOT, under any circumstances, open the entire station to antagonists. If anything, this makes the antagonist's job harder since they no longer have a valid way of quickly downing multiple people trying to put them down
Why yes it does! I go on to explain why. You really should read the entirety of what I have written before going off on it. The antagonists have plenty of ways to quickly down multiple people. The revolver, the double sided energy sword, magic missile, nuke op weaponry, changeling stings and vampire powers, I could go on.

>For one, this is a GOOD thing that a Traitor would remotely consider the Revolver and the Energy Sword viable alternatives to other equipment, since right now they're flamebait and newbie bait.
No, no, no! You are wrong and I have proof this time. You keep saying things that are unfalsifiable (yet so obviously false to anyone who plays the game) but to say these items are not popular is a lie. You can look at the statistics here. The energy sword remains the 3rd most popular traitor item while the revolver the 7th. Unless you want to make the argument that the traitor styles of /tg/ and /vg/ are SO vastly different, in which case you are grossly uninformed.

http://www.ss13.eu/tgdb/latest_stats.html

/obj/item/weapon/melee/energy/sword (4TC) (Total):
/obj/item/weapon/gun/projectile/revolver (6TC) (Total):

And even if what you said was true? So what? What they are instead elevated to be are incredibly powerful weapons, more powerful then a taser could hope to be. 

>clearly the energy crossbow should not keep its instant stun on hit
So it would never be picked and is now useless. Congratulations.

>And CR20s give you a large window of time to react and fight back while tasers put you out of the fight for 10 seconds, or forever if anyone can even bother
I'm not sure what all that nonsense before that had to do with what I'm saying, but this statement is patently false. Once you are shot with the CR20 you are stunned. You are then shot 4 more times while lying on the ground. You are now in crit. You then die. That is how nuke ops rounds go. You'd know if you play. 

>Wizards are supposed to be overpowered since they are a one-man-army antag by design
And now they are even more powerful. Good job.

>Death stings have nothing to do with stuns and are delivered in a stealthy manner. 
Death stings stun you through being repeatedly knocked down and then kill you. Do you even play the game? 

>Factoring everything above, fuck no. This makes it a bit easier for antags, of course, but then Security will actually get robust and use some real fucking weaponry, or will learn to not just stand in the middle of a hallway soaking bullets and lasers
No, it makes ridiculously easier for antags. Security does not by default have access to lethal weapons. They would need to go into the armory and get them. Now security is FORCED to use lethals. If you cannot think of at least 5 problems this would cause I'd say you're an idiot. I'll give you some hints though: AI, Cyborg, Admins, Lawful Arrest, Misunderstandings.
>That claim is ridiculous, SS13, at its absolute base, was designed as a space station simulation with atmospherics in mind.
Don't be an idiot. The entire game, every single game decision involving weaponry, combat, antagonists, etc is based on the stun system that has been in place for almost an entire decade. Why you want to change a system that has been in place since years you have started playing is beyond me. Why you want to handle the massive amounts of changes, balance fixes, bugs, etc 
is again, lost on me. All for a system that MAY have some benefits (though nobody here has disclosed any) to solve some non existent problem

>We know what we are doing. The amount of time where people have burst in holy rage due to us changing major systems and claiming everything was over, and subsequently, within a week or so, have gotten used to the changes and developed a new meta, are countless. No-one here is out of touch but you, for claiming all of this

You have absolutely no idea what you are doing. You are making drastic changes for no reason. You are not even bothering to consider that it may not be a good idea.

**Comment by Probe1** [comment]
malf ai is a weird game mode to discuss balance about because it depends on a tremendous amount of variables, some of which are measurable and some of which are not.  will the heads or even just regular players recognize a malf AI and know how to defeat it?  will the captain be a scrub and not have the pinpointer?  how many borgs does the AI have?  which modules did it pick?  will people work together against the AI or will they flee to the asteroid?

**Comment by Uqualm** [comment]
>malf ai is a weird game mode to discuss balance about

No it's not. The malfunctioning AI used to be 1000 times more powerful then it is now. You are a self admitted new player and would not remember.

**Comment by VelardAmakar** [comment]
breaking news: sarca is a dumb fuckboy

**Comment by SangriaMonkey** [comment]
Ayy

**Comment by VelardAmakar** [comment]
how does it feel to be white, pathetic, and in denial of how much you desire male affection

**Comment by NigglyWiggly** [comment]
People who don't play SS13 arguing with people who don't play SS13 about changing SS13

Fun!

**Comment by SangriaMonkey** [comment]
I've played daily for about three years.

**Comment by eswordthecat** [comment]
Adding 1 to comments.

**Comment by Uqualm** [comment]
Doesn't matter, I'm right and you know it 

:^).

Unless you got some facts then lay em down.

**Comment by VelardAmakar** [comment]
i don't care either way, i think stuns should change at some point but now is not the time, and this discussion won't lead to anything

i'm just taking a chance to badmouth sarca

**Comment by SangriaMonkey** [comment]
It's not even discussion. It's more like two sides reading off terrorist demands.

**Comment by NigglyWiggly** [comment]
>I've played daily for about three years.

Not you, silly

**Comment by xevonix** [comment]
Stuns are fine now, don't make it harder for Sec to catch murderfuckers armed with ebows.
Yeah, a close-up stun is pretty cheap but if you had your goddamned suit sensors turned to maximum and managed to atleast call out "HELP" beforehand then the AI or Medical can always find you.

**Comment by ComicIronic** [comment]
>Stuns are fine now, don't make it harder for Sec to catch murderfuckers armed with ebows.

Why not?
I accept that sec should be able to catch criminals, but why should that mean any sort of dominance over combat?

And the ultimate issue is not what happens after combat is over, it's the combat itself.

**Comment by Shadowmech88** [comment]
I figure I might as well throw my suggestion into the ring, though it's not like we'll ever reach enough of a consensus to actually change anything.
I was picturing making it to where getting hit by a stun shot causes you to drop your things and fall over like it does now, but you're able to pick your weapon back up while you're on the ground and return fire, in order to give the person who was stunned a bit of a fighting chance. Additionally, I'd make it so that stuns don't stack. You wouldn't be able to re-stun someone until the stun wears off and they get back up, however stunning the person while they're down will cause them to drop their weapon again if they'd picked it back up.
This won't hurt security officers in a major way because they always carry cuffs, and would be able to cuff one or even two people before their stuns wear off and they're able to get up and run. He could also prevent them from running by taking aim at them while they're on the ground (it'd be nice in general to see the "take aim" feature used more often.)
This will however, require antags to take a little more care with preparation, requiring them to have cable cuffs ready if they want to truly subdue someone with no chance of them escaping, since they can't just keep shooting them as they drag them. In the event of a spur-of-the-moment fight, the antag will have to think fast and either fashion cable cuffs then, kill their victim, break their legs, or any number of things more interesting than simply standing over their victim and tasing them repeatedly while they think of what to do next.
I also think that stun prods should work this way as well, given their makeshift nature. Bona fide stun batons are fine the way they are, given that they're both melee weapons and Security-only.

What do you guys think?

**Comment by Probe1** [comment]
I think if someone was carrying a weapon and could pick it up when they were on the ground I would just laser them into crit rather than deal with getting stunned myself and having him kill me/get away.

**Comment by Shadowmech88** [comment]
Fair point.
What about the ranged-stuns-not-stacking proposal?

**Comment by The-Whargoul** [comment]
"I am personally a fan of Skowron's suggestion, which could use pain in interesting ways to allow sec officers to use tasers more effectively. It would require sufficient pain to the chest to knock someone over, for example, but targeting their legs could slow them down or targeting their hands could disarm them. This also has the benefit of integrating well with disarm pushes, since it would no longer be possible to just shove and fighting would involve real fistwork."

I really like this idea, makes combat more interactive. However, with such a change you would have to rework the whole combat system, or at least take all factors into account.

**Comment by clusterfack** [comment]
Well of course, that's the point of this issue discussion, talking about a total rewrite of the way combat works currently. No sane coder would want to rework a whole system if they knew it would dead on arrival though (other than goofball).

**Comment by MercuryDoll** [comment]
I like the idea of stamina. Making runspeed and stun potential dependant on your stamina gives combat some strategic element, as you want to manage your stamina and not run out of breath. Getting hit by a taser close range instantly drains about 90% of stamina, if he was running it would effectively stun the target, as completely running out of stamina immobilizes the target (but not muting him, people should get their chance of shouting their last breath). And even if he was at full stamina it would have drained most of his stamina, slowing him down to a crawl. The taser projectile loses charge over distance though, so a long-range taser shot may only drain a portion of the victim's stamina. This won't completely stun the victim, but slow him down significantly, depending on how much stamina he has left, as even simply running through the hallway itself saps stamina. Stunbaton could be similar, sapping about 80% stamina on hit (60% if thrown). Hunger will have a regenerating effect on stamina if you're satiated. Painkiller and other drugs can have an effect on stamina too, temporarily boosting your endurance.
The effect of stamina would probably exponential, a change in speed noticable at about 50% and getting slower the more you run out of breath.
![staminacurve](http://i.imgur.com/MNUBKqL.png)

**Comment by Probe1** [comment]
Old discussion.
